### PR TITLE
[16] Rendered the related product list on the console with JavaScript

### PR DESCRIPTION
**Pull request by elizabethbusola**
## Task: `To render the product list on the console with Javascript
- I created an array that has three objects with each object having three properties for the productlist that was created for the product  page. 

image

### CODE REVIEW

File: submissions/busolaAdedoyin/javascript/index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SignUp Page</title>
+    <link href="../css/index.css" type="text/css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Arimo&display=swap" rel="stylesheet">
+    <script src="index.js"></script>
+</head>
+<body>
+
```
Comment (adaugochi) [inline submissions/busolaAdedoyin/javascript/index.html:12]: From line 12 - 105 is irrelevant for this task. All that you needed to do here is link the js file

**Review by adaugochi — CHANGES_REQUESTED** [review]
Also, adding new images is irrelevant. work with the existing images you have. Except you want to update the previous ones you have

File: submissions/busolaAdedoyin/javascript/index.html (hunk 1 above)
Comment (adaugochi) [reply submissions/busolaAdedoyin/javascript/index.html:12]: also, there are no closing `body` and `html` tags here

Comment (elizabethbusola) [reply submissions/busolaAdedoyin/javascript/index.html]: > From line 12 - 105 is irrelevant for this task. All that you needed to do here is link the js file

@adaugochi changes have been made and images deleted

File: submissions/busolaAdedoyin/javascript/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+
+var x = "";
+var productList = [
+    Fabric1  = {
```
Comment (adaugochi) [inline submissions/busolaAdedoyin/javascript/index.js:4]: there is no need for naming this object. Simply declare the object. Remove the Fabric1 same goes to others

File: submissions/busolaAdedoyin/javascript/index.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+
+var x = "";
```
Comment (adaugochi) [inline submissions/busolaAdedoyin/javascript/index.js:2]: You already declare x in the for-in loop. you can remove  it

File: submissions/busolaAdedoyin/javascript/index.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SignUp Page</title>
+    <link href="../css/index.css" type="text/css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Arimo&display=swap" rel="stylesheet">
```
Comment (adaugochi) [inline submissions/busolaAdedoyin/javascript/index.html:8]: remove font. It is not needed

File: submissions/busolaAdedoyin/javascript/index.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SignUp Page</title>
+    <link href="../css/index.css" type="text/css" rel="stylesheet">
```
Comment (adaugochi) [inline submissions/busolaAdedoyin/javascript/index.html:7]: Remove css, it is not needed

File: submissions/busolaAdedoyin/javascript/index.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+        productPrice:8750
+    }
+];
+ for(var i = 0; i < productList.length; i++){
+    for(var  x in productList[i]){
+        console.log(productList[i]['productName'] + " with " + productList[i]['imageUrl'] + " is sold for  " + productList[i]['productPrice'])
+    };
+
+ }
+
```
Comment (adaugochi) [inline submissions/busolaAdedoyin/javascript/index.js:26]: Remove spaces

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by elizabethbusola** [comment]
@adaugochi All the changes requested have been made.

File: submissions/busolaAdedoyin/javascript/index.js [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+        imageUrl: 'img/lacefab.jpg',
+        productPrice: 12500,
+    },
+    {
+        productName: 'Polish Fabric',
+        imageUrl:'img/polishfab.jpg',
+        productPrice:8750
+    }
+];
+ for(var i = 0; i < productList.length; i++){
```
Comment (ToluAkin) [inline submissions/busolaAdedoyin/javascript/index.js:19]: Use one for loop @elizabethbusola

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/busolaAdedoyin/javascript/index.js (hunk 7 above)
Comment (elizabethbusola) [reply submissions/busolaAdedoyin/javascript/index.js]: @ToluAkin I have removed one for loop.

**Review by ToluAkin — APPROVED** [review]

**Pull request merged**